src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -1025,6 +1041,22 @@ public function updateQuietly(array $attributes = [], array $options = [])
         return $this->fill($attributes)->saveQuietly($options);
     }
 
+    /**
+     * Update only the missing values of the model in the database.
+     *
+     * @param  array  $attributes
+     * @param  array  $options
+     * @return bool
+     */
+    public function updateMissing(array $attributes = [], array $options = [])
```

Comment:
Maybe `updateBlanks` is a better name as we have the filled() & blank() helpers, this way we can follow a "convention"

Reply:
Thanks, I updated the name to `updateBlanks`